package.json

```diff
@@ -132,5 +138,20 @@
       "<rootDir>/jestMockSetup.js"
     ],
     "cacheDirectory": ".jest/cache"
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "$root$": false,
+      "@fortawesome/fontawesome-svg-core": false,
+      "@fortawesome/fontawesome-svg-core>@fortawesome/fontawesome-common-types": false,
+      "@fortawesome/free-regular-svg-icons": false,
+      "@fortawesome/free-regular-svg-icons>@fortawesome/fontawesome-common-types": false,
+      "@fortawesome/free-solid-svg-icons": false,
+      "@fortawesome/free-solid-svg-icons>@fortawesome/fontawesome-common-types": false,
+      "@hathor/wallet-lib>level>classic-level": false,
+      "@lavamoat/preinstall-always-fail": false,
+      "@sentry/react-native>@sentry/cli": false,
+      "ttag-cli": false
+    }
```

Comment:
Why did you set this? Is this a test or these are real validations?

Reply:
These are real validations, it will disallow those libraries (which are the ones with postinstall scripts) to execute the postinstall

Reply:
Is there any configuration to `disallowAll`?

Reply:
It actually denies all scripts by default, this config was auto generated by using `allow-scripts auto`, I think it's nice to have so we know which dependencies had scripts.

Here is the output of `allow-scripts list`:

```
❯ npx allow-scripts list

# allowed packages with bin scripts
  (none)

# allowed packages with lifecycle scripts
  (none)

# disallowed packages with lifecycle scripts
- $root$ [1 location(s)]
- @fortawesome/fontawesome-svg-core [1 location(s)]
- @fortawesome/fontawesome-svg-core>@fortawesome/fontawesome-common-types [1 location(s)]
- @fortawesome/free-regular-svg-icons [1 location(s)]
- @fortawesome/free-regular-svg-icons>@fortawesome/fontawesome-common-types [1 location(s)]
- @fortawesome/free-solid-svg-icons [1 location(s)]
- @fortawesome/free-solid-svg-icons>@fortawesome/fontawesome-common-types [1 location(s)]
- @hathor/wallet-lib>level>classic-level [1 location(s)]
- @lavamoat/preinstall-always-fail [1 location(s)]
- @sentry/react-native>@sentry/cli [1 location(s)]
- ttag-cli [1 location(s)]
```